docs/developer/metadata.mdx

```diff
@@ -113,6 +113,15 @@ type Mutation {
 }
 ```
 
+:::note
+For `Order` and `Checkout` as the `id` input value, you can provide either `token` or `id` value.
+:::
+
+:::warning
+In the near future, changing `Order` metadata will be available only with the use of `token`.
```

Comment:
How about to specify version number?

Reply:
I can add. @maarcingebala which version?

Reply:
We want to do that before release stable 3.0, and the docs are targeting the 3.0 version. Maybe we don't have to mention that it will change, but at this point we should say that with order and checkout type you should use token as identifier, while for other types you should use GraphQL ID?